rskj-core/src/main/java/org/ethereum/datasource/KeyValueDataSourceUtils.java

```diff
@@ -109,9 +109,6 @@ public static void validateDbKind(DbKind currentDbKind, String databaseDir, bool
         if (prevDbKind != currentDbKind) {
```

Comment:
Is this check really needed now, or can we simply check `if (databaseReset)` to create/update the file?

Reply:
We can simply check `if (databaseReset)`